vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -24,6 +38,7 @@ type Config struct {
 	StdinOnce       bool                  // If true, close stdin after the 1 attached client disconnects.
 	Env             []string              // List of environment variable to set in the container
 	Cmd             strslice.StrSlice     // Command to run when starting the container
+	Healthcheck     *HealthConfig         `json:",omitempty"` // Command to run to check container is healthy
```

Comment:
On 13 May 2016 21:20, "Stephen Day" <notifications@github.com> wrote:
>
> In vendor/src/github.com/docker/engine-api/types/container/config.go:
>
> > @@ -24,6 +38,7 @@ type Config struct {
> >  StdinOnce       bool                  // If true, close stdin after
the 1 attached client disconnects.
> >  Env             []string              // List of environment variable
to set in the container
> >  Cmd             strslice.StrSlice     // Command to run when starting
the container
> > + Healthcheck     *HealthConfig         `json:",omitempty"` // Command
to run to check container is healthy
>
> @justincormack I would expect support for multiple HEALTHCHECK clauses in
a Dockerfile. Any reason not to?
>

How would you then eg override one of them? Maybe it should be an array of
checks so you have to replace all of them?